haskell/src/Client/Contract.hs

```diff
@@ -0,0 +1,73 @@
+-- SPDX-FileCopyrightText: 2020 tqtezos
+-- SPDX-License-Identifier: MIT
+
+module Client.Contract
+  ( stablecoinContractContent
+  , parseStablecoinContract
+  , mkInitialStorage
+  ) where
+
+import Data.FileEmbed (embedStringFile)
+import qualified Data.Map.Strict as Map
+import Michelson.Runtime (parseExpandContract)
+import Michelson.Text (MText)
+import Michelson.Typed (BigMap(BigMap))
+import qualified Michelson.Untyped as U
+import Tezos.Address (Address)
+import Util.Named ((.!))
+
+import Lorentz.Contracts.Stablecoin (Storage, mkTokenMetadata)
+
+stablecoinContractContent :: Text
+stablecoinContractContent = $(embedStringFile "../ligo/stablecoin.tz")
+
+parseStablecoinContract :: MonadThrow m => m U.Contract
+parseStablecoinContract =
+  either throwM pure $
+    parseExpandContract
+      (Just "../ligo/stablecoin.tz")
```

Comment:
Since we are also using the contract in tests, I would define `stablecoinPath :: FilePath` in the library, export it and use in tests, so that the path does not appear in multiple places.

Reply:
Done